modules/ROOT/partials/configuration/forced_root_block_attrs.adoc

```diff
@@ -5,7 +5,6 @@ This option enables you specify attributes for the xref:forced_root_block[`force
 
```

Comment:
The line above contains an xref that refers to things in other files, it's better to include the full ref:
```xref:forced_root_block[`forced_root_block`]```